### DISCUSSION TITLE
Live Reload Functionality for Real-Time Updates

### DISCUSSION THREAD

**Discussion by lucasjose501** [opcodesio/log-viewer#289] [Ideas]
Hello!

I've been using this package since it was released and it's been super helpful and I'd like to leave this Idea here.

The idea is to have a toggle option to update logs in "real-time" every few seconds. It's a feature that can save us a lot of manual refreshing.

What do you all think? Could this be a valuable addition to the Log Viewer package?

Looking forward to your feedback,

Cheers.

**Comment by arukompas** [comment]
That is indeed a great feature to have! I'll keep it in mind for our next release :)
